pokemongo_bot/cell_workers/pokemon_catch_worker.py

```diff
@@ -221,6 +221,23 @@ def _pokemon_matches_config(self, config, pokemon, default_logic='and'):
             'iv': False,
         }
 
+        candies = inventory.candies().get(pokemon.pokemon_id).quantity
+        threshold = pokemon_config.get('candy_threshold', 400)
```

Comment:
candy_threshold should default to false since this code will be triggered when config not changed for candy_threshold.